### DISCUSSION TITLE
Split unclassified roads and residential roads in the default.render.xml for better visual control

### DISCUSSION THREAD

**Discussion by peterelli** [osmandapp/OsmAnd#12000] [Maps Related]
Hello developers,

# 🚀 feature request

in rendering unclassified roads and residential roads can only have one color. Residential roads are mainly in cities where unclassified roads are in the countryside. When you want to emphasize the unclassified roads with more intense color you have the map painted in this color because the many residential roads. 

### Describe the solution you'd like

Please split these two in the render.xml to make color changes more easy. In the past these two were separate but it is changed now.

Thank you very much.

**Comment by vshcherb** [comment]
Converted request to a discussion to have wider opinion. Each map style has obviously own vision how to identify roads and actually different authors. Default map style is trying to use as less colors as possible to give wider range for modifiers, I think unclassified and residential slightly differ by the width actually. Though I can't say much about other rendering styles.

**Reply by peterelli** [reply]
It is not necessary to have different colors for the two road types as standard, only an easy opportunity to change these in the default.render.xml if wanted.
